GPU/Debugger/Record.cpp

```diff
@@ -619,7 +619,6 @@ void ClearCallback() {
 
 static void FinishRecording() {
 	// We're done - this was just to write the result out.
-	Path filename = WriteRecording();
```

Comment:
This can't be good.  Certainly broke frame dumps.

-[Unknown]

Reply:
Yup, already reverted